employee-rostering-gwtui/src/main/java/org/optaweb/employeerostering/gwtui/client/tenant/NewTenantForm.java

```diff
@@ -187,6 +187,6 @@ private static int valueAsInt(HTMLInputElement inputElement) {
     }
 
     private static native String getSystemTimezone() /*-{
-        return Intl.DateTimeFormat().resolvedOptions().timeZone;
-    }-*/;
+                                                     return Intl.DateTimeFormat().resolvedOptions().timeZone;
+                                                     }-*/;
```

Comment:
This is unexpected. The original format was correct. I recommend to revert this change.

Reply:
This was caused from the Eclipse formatter formatting the code (since it is a comment and I didn't set the formatter on/off tags yet since my latest Eclipse reboot) and the IDEA formatter **not** reformatting the code. There is a minor issue with the Pull Request, where the Schedule Start Date field in the new Tenant Form is not pre-populated when the form is created, but is when you click on it. None of the other forms have this issue. Also an issue in Firefox where the
"Today" button has no background (not an issue in Chrome). Will fix the formatting in a commit that also fixes those issues.